.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,61 +1,40 @@
 ---
 name: ✨ Feature request
 about: Suggest an idea for this project
-
 ---
-
-<!---
-Verify first that your issue/request is not already reported on GitHub.
-THIS FORM WILL BE READ BY A MACHINE, COMPLETE ALL SECTIONS AS DESCRIBED.
-Also test if the latest release, and devel branch are affected too.
-ALWAYS add information AFTER (OUTSIDE) these html comments.
-Otherwise it may end up being automatically closed by our bot. -->
+<!--- Verify first that your feature was not already discussed on GitHub -->
+<!--- This form is processed by Ansibot (machine), complete *all* sections as described -->
 
 ##### SUMMARY
-<!--- Explain the problem briefly -->
+<!--- Describe the new feature/improvement briefly below -->
 
 ##### ISSUE TYPE
- - Feature Idea
+- Feature Idea
```

Comment:
We need to keep the instructions to a minimum IMO.
Keep it as short and simple as possible.
And I don't think a lot of people would be removing it.
(In a future rendition, this would be in the issue title. See original commit)

Reply:
+1 to keeping this short, need to avoid the "wall of text"

Reply:
It's fairly rare that people change the `Issue Type`, if they do the Bot lets them know.

I don't think we need to change anything here

Reply:
Not really, bot only checks the presence of titles and doesn't react if a person removes all types and leaves no text there at all.

Reply:
cc @jctanner